commands/issue/create/issue_create.go

```diff
@@ -119,11 +121,27 @@ func createRun(opts *CreateOpts) error {
 		return err
 	}
 
+	remotes, err := opts.Remotes()
+	if err != nil {
+		return err
+	}
+	repoRemote, err := remotes.FindByRepo(repo.RepoOwner(), repo.RepoName())
+	if err != nil {
+		return err
+	}
+
```

Comment:
The tests points out an important error that's likely to happen due to the usage of the `remotes.FindbyRepo`:
```sh
=== Failed
=== FAIL: commands/issue/create Test_IssueCreate (0.35s)
    issue_create_test.go:76: -t myissuetitle -d myissuebody -l test,bug --weight 1 --milestone 1 --linked-mr 3 --confidential --assignee testuser -R glab-cli/test

Error: no matching remote found
```
https://gitlab.com/profclems/glab/-/jobs/938231788

Looking at it carefully, it is not really related to this PR. Since everything looks good, I will merge and open a new PR to fix this by changing the signature of the `MilestonePrompt` function to use `repo` instead of `repoRemote`
https://github.com/profclems/glab/blob/555afc9098aed6b471a100b419273493745c116b/commands/cmdutils/cmdutils.go#L178